src/parser/parsers/naive.ts

```diff
@@ -1368,7 +1370,7 @@ export function readFieldSpecificationList<S extends IParserState = IParserState
     );
     const fields: Ast.ICsv<Ast.FieldSpecification>[] = [];
     let continueReadingValues: boolean = true;
-    let maybeOpenRecordMarkerConstant: Ast.IConstant<Ast.MiscConstantKind.Ellipsis> | undefined = undefined;
+    let isOnOpenRecordMarker: boolean = false;
```

Comment:
Bug fix where the open record constant marked as the child of the inccorrect node.